authors/apps/authentication/views.py

```diff
@@ -105,3 +112,73 @@ def update(self, request, *args, **kwargs):
         serializer.save()
 
         return Response(serializer.data, status=status.HTTP_200_OK)
+
+class PasswordResetView(APIView):
+
+    def post(self, request):
+        if 'email' not in request.data:
+            raise exceptions.ValidationError("Please provide an Email Address")
+        email=request.data["email"]
+        try:
+            if not email:
+                raise exceptions.ValidationError("Please fill in the email field")
+
+            if not re.match(r"(^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9-]+\.[a-zA-Z0-9-.]+$)", email):
+                raise exceptions.ValidationError("Invalid Email Address")
+
+            if not User.objects.filter(email = email).first():
+                raise exceptions.ValidationError("This Email Address is not attached to any account")
+
+            serializer = URLSafeTimedSerializer(os.environ.get("SECRET_KEY"))
+            token = serializer.dumps(email, salt=os.environ.get("SECURITY_PASSWORD_SALT"))
+            
+            BASE_URL = os.environ.get("BASE_URL", "http://127.0.0.1:8000")
+            reset_link = "{}/api/users/change-password/{}/".format(BASE_URL,token)
+            recipient = [email]
+            sender = 'habibsentongo@gmail.com'
```

Comment:
@HabibSentongo I suggest change the sender email to the group email - which is saved in this env var **EMAIL_HOST_USER**

Reply:
Thank you, I have corrected this mistake.